bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -116,56 +116,56 @@ <h3 class="primary-title-sub">{{ _('Fast for good.') }}</h3>
       <div class="mzp-l-card-hero">
         {{ card_block(
             class='mzp-c-card-large',
-            tag_label=_('Internet health'),
+            tag_label=_('Podcast'),
             media_icon='mzp-has-video',
-            title=_('The Today Show: Firefox and user control'),
-            ga_title='The Today Show: Firefox and user control',
-            desc=_('Mozilla founder Mitchell Baker talks about internet security, user control and how our Facebook Container can help.'),
-            image_url='home/2018/today-show.jpg',
+            title=_('IRL season 3: The grand bargain'),
+            ga_title='IRL season 3: The grand bargain',
+            desc=_('You click “accept” every day, but often don’t know what you’re giving away. Is it a fair trade, or should you demand a better bargain?'),
+            image_url='home/2018/cards/irl-season-3.png',
             include_highres_image=True,
             aspect_ratio='mzp-has-aspect-16-9',
-            link_url='https://blog.mozilla.org/internetcitizen/2018/06/06/the-today-show-mozilla-firefox-facebook-container/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card',
+            link_url='https://irlpodcast.org/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card',
+            video_url='https://assets.mozilla.net/video/mozilla-home/irl-trailer'
         )}}
         {{ card_block(
-            tag_label=_('Innovation'),
-            title=_('The design of Firefox Reality'),
-            ga_title='The design of Firefox Reality',
-            desc=_('Firefox Reality is bringing the wider web to stand-alone AR and VR headsets so that we can see ourselves, each other and the world better.'),
-            image_url='home/2018/mixed-reality.png',
+            tag_label=_('News'),
+            title=_('The New York Times reviews Firefox'),
+            ga_title='The New York Times reviews Firefox',
+            desc=_('The New York Times declares Firefox is back, and it’s time to give it a try.'),
+            image_url='home/2018/cards/nyt.png',
```

Comment:
Greetings, code reviewer! Yes, this image features the old Firefox logo which is unfortunate. This has been discussed with the folks in charge of curating the home page content and they have chosen to stick with the same image that NYT created for this story, despite it not using an up-to-date logo.